runtime-infra/fragments/ecs-service.yaml

```diff
@@ -1151,6 +1161,108 @@ Resources:
                 Effect: Allow
                 Resource: "*"
 
+  ###                            Event adn Alarm                           ###
+  ############################################################################
+
+  EventRule:
+    Type: AWS::Events::Rule
+    Properties:
+      Name: !Sub "ECSStoppedTasksEvent-${ECSService.Name}"
```

Comment:
Use `${MicroServiceUniqueName}` instead of `${ECSService.Name}`

Reply:
done